ecs-cli/modules/cli/compose/entity/entityType/entityType.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package entityType
```

Comment:
**type** is used in go, don't think I can name it as type, but I can name it as types. What do you think? Right now is entityType.Type, but with the change it'll be types.EntityType